css/.scss-lint.yml

```diff
@@ -219,13 +220,19 @@ linters:
     SpaceAfterVariableName:
         enabled: true
 
+    # Checks for space around operators on values.
+    # Configure: `one_space` or `no_space`
+    SpaceAroundOperator:
+        enabled: true
+        style: one_space
```

Comment:
This is for things like `$unit*2` vs `$unit * 2`?

Do we have a preference for one over the other?